src/commands/Games/trivia.ts

```diff
@@ -0,0 +1,88 @@
+import { SkyraCommand, SkyraCommandOptions } from '@lib/structures/SkyraCommand';
+import { ApplyOptions } from '@skyra/decorators';
+import { KlasaMessage, KlasaUser } from 'klasa';
+import TriviaManager, { CATEGORIES, QuestionType, QuestionDifficulty, QuestionData } from '@utils/Games/TriviaManager';
+import { decode } from 'he';
+import { shuffle } from '@utils/util';
+import { MessageEmbed, MessageCollector } from 'discord.js';
+
+@ApplyOptions<SkyraCommandOptions>({
+	cooldown: 5,
+	description: language => language.tget('COMMAND_TRIVIA_DESCRIPTION'),
+	extendedHelp: language => language.tget('COMMAND_TRIVIA_EXTENDED'),
+	usage: '(category:category) [boolean|multiple] [easy|hard|medium] [duration:int{30,60}]',
+	usageDelim: ' '
+})
+export default class extends SkyraCommand {
+
+	// eslint-disable-next-line @typescript-eslint/explicit-member-accessibility
+	#channels = new Set<string>();
+
+	public async run(message: KlasaMessage, [category, questionType = undefined, difficulty = undefined, duration = 30]: [number, QuestionType?, QuestionDifficulty?, number?]) {
+		if (this.#channels.has(message.channel.id)) throw message.language.tget('COMMAND_TRIVIA_ACTIVE_GAME');
+
+		this.#channels.add(message.channel.id);
+		await message.sendLocale('SYSTEM_LOADING');
+		const data = await TriviaManager.getQuestion(category, difficulty, questionType);
+		const possibleAnswers = questionType === QuestionType.BOOLEAN
+			? ['True', 'False']
+			: shuffle([data.correct_answer, ...data.incorrect_answers].map(ans => decode(ans)));
+		const correctAnswer = decode(data.correct_answer);
+
+		await message.sendEmbed(this.buildQuestionEmbed(message, data, possibleAnswers));
+		const filter = (msg: KlasaMessage) => {
+			const num = parseInt(msg.content, 10);
```

Comment:
We always use `Number`, since `parseInt('11aa');` returns `11` instead of `NaN`.